src/components/menu/menu.d.ts

```diff
@@ -5,7 +5,10 @@ type menuType = "light" | "dark" | "white" | "black";
 interface MenuContextProps {
   menuType: menuType;
   isFirstElement: boolean;
-  handleKeyDown: (ev, submenuOpen) => void;
+  handleKeyDown: (
+    ev: React.KeyboardEvent<HTMLAnchorElement | HTMLButtonElement>,
```

Comment:
After conversation on slack we've agreed that the proposed change is correct as it is passed to MenuItems that can render as either a `HTMLAnchorElement` or `HTMLButtonElement` element so needs to support and event from both types rather than one or the other